src/Integrations/Mailer/Mailer.php

```diff
@@ -405,4 +405,28 @@ private function prepareFiles(array $files): array
 
 		return $output;
 	}
+
+	/**
+	 * Clean up form details.
+	 *
+	 * @param array<string, mixed> $formDetails Data passed from the `getFormDetailsApi` function.
+	 *
+	 * @return array<string, mixed>
+	 */
+	private function cleanUpFormDetails(array $formDetails): array
+	{
+		$list = [
+			UtilsConfig::FD_FIELDS,
+			UtilsConfig::FD_FIELDS_ONLY,
+			UtilsConfig::FD_ICON,
+		];
+
+		foreach ($list as $item) {
+			if (isset($formDetails[$item])) {
+				unset($formDetails[$item]);
+			}
+		}
+
+		return $formDetails;
```

Comment:
```suggestion
		$list = [
			UtilsConfig::FD_FIELDS,
			UtilsConfig::FD_FIELDS_ONLY,
			UtilsConfig::FD_ICON,
		];

		return array_diff_key($formDetails, array_flip($list));
```

Should work the same way.